astropy/cosmology/tests/test_connect.py

```diff
@@ -9,16 +9,23 @@
 from astropy.cosmology import Cosmology, w0wzCDM
 from astropy.cosmology.connect import readwrite_registry
 from astropy.cosmology.io.tests import (
-    test_cosmology, test_ecsv, test_json, test_mapping, test_model, test_row, test_table, test_yaml)
+    test_cosmology, test_ecsv, test_html, test_json, test_mapping, test_model, test_row, test_table,
+    test_yaml)
 from astropy.table import QTable, Row
+from astropy.utils.compat.optional_deps import HAS_BS4
 
 ###############################################################################
 # SETUP
 
 cosmo_instances = cosmology.realizations.available
 
 # Collect the registered read/write formats.
-readwrite_formats = {"ascii.ecsv", "json"}
+readwrite_formats = {
```

Comment:
Probably good to describe what these options are (maybe even `namedtuple`).

Reply:
I've added a description.